### DISCUSSION TITLE
Status multiple replies

### DISCUSSION THREAD

**Discussion by adrianomic** [mastodon/mastodon#16997] [Q&A]
I would like to extend status to be able to reply to any reply.
For now the we can reply only to original status like reddit and twitter
We can only do boost and favorite actions to replies.
Reblog(boost) is also status, right?
I would like to add root_id which points the original status.
Do we already have that field in Status or do I need to add?

**Comment by Gargron** [comment]
No, you can reply to any reply

**Reply by adrianomic** [reply]
Can you show me reply of reply example?

**Reply by Gargron** [reply]
Here is someone replying to a reply. https://mastodon.social/@trwnh/107279403476084031

**Reply by adrianomic** [reply]
Thanks, but I don't understand this UI. In reddit, I can see easily which one is reply of which thing.

BTW, How to get only original posts easily?
Let me know if I think in correct way.
In all timelines, I would like to return only original post ids from backend. Original post ids that can be referred to root posts.
Then when showing original post in react side, I would like to front-end to send api request to get all descendants of this root posts.
We can think this as tree structure. So I want to get all descendants from root id easily. I decided to add root_id to every status in table.
If we add this root_id, descendant status can find root status easily and root status can find all descendants easily
